test/tests/valid/export.test.js

```diff
@@ -17,7 +17,7 @@ describe('VALID: Export Script', function () {
 
     it('Example: should exist without attributes or errors', function () {
         expect(data[0]?.example).to.exist;
-        expect(data[0].example.attributes).to.be.empty;
+        expect(data[0].example.attributes).to.not.be.empty;
         expect(data[0].example.errors).to.be.empty;
     });
```

Comment:
Wait so why is example found here? its not exported in `export.valid.js`?

Reply:
Yep it is. Exported here https://github.com/playcanvas/attribute-parser/pull/8/files#diff-9916309e65df937dbb358e1fa5188b9971ee43562737ca027fa27252501bffe7L19